src/sonic-ctrmgrd/ctrmgr/ctrmgr_tools.py

```diff
@@ -0,0 +1,152 @@
+#! /usr/bin/env python3
+
+import os
+import sys
+
+sys.path.append("tests")
+import docker
+import argparse
+
+import json
+import syslog
+
+from swsscommon import swsscommon
```

Comment:
Why break these out into different groups? Per PEP8 (and ignoring the `sys.path.append("tests")` line for now, per my question above), the imports should be organized as follows:

```python
import argparse
import json
import os
import sys
import syslog

import docker
from swsscommon import swsscommon
```

Reply:
done. Here  and as well in ctrmgr/container, I  moved  "import docker" from the ordered set above, as suggested here.